cobeylab/imprinting
# Added documentation files

This should resolve #29.

To test, run:

```
library(devtools)
build()
install()
test()
library(imprinting)

?get_imprinting_probabilities
```

The final line should return help documentation for `get_imprinting_probabilities()`

Furthermore, in the Rstudio command line, if you type `?imprinting::`, it should pull up an interactive menu showing a list of functions exported by the package.
